distage/distage-framework-docker/src/test/scala/izumi/distage/testkit/docker/DistageTestDockerBIO.scala

```diff
@@ -30,10 +34,32 @@ final class DistageTestDockerBIO extends DistageBIOSpecScalatest[IO] {
 
   override protected def config: TestConfig = {
     super.config.copy(
-      memoizationRoots = Set(
-        DIKey.get[PgSvcExample],
-//        DIKey.get[DynamoDocker.Container],
-//        DIKey.get[PostgresDocker.Container],
-      ))
+      memoizationRoots = Set(DIKey.get[PgSvcExample]),
+      parallelTests = true,
+      parallelEnvs = true,
+      testRunnerLogLevel = Log.Level.Debug
+    )
   }
 }
+
+final class DistageTestDockerBIO1 extends DistageTestDockerBIO { override protected def config: TestConfig = DistageTestDockerBIO.randomEnv(super.config) }
```

Comment:
@Caparow You can put this once into the super class, don't have to repeat always

Reply:
done